Mathlib/Lean/Name.lean

```diff
@@ -56,7 +56,14 @@ def Lean.Name.decapitalize (n : Name) : Name :=
     | n       => n
 
 /-- Whether the lemma has a name of the form produced by `Lean.Meta.mkAuxLemma`. -/
```

Comment:
The docstring of this and the next declaration need to be updated, right? Since `_proof_i` decls are not generated by `mkAuxLemma`?